src/Metrics.h

```diff
@@ -154,9 +154,23 @@ class Quantile final : public Metric
         _quantile.update(value);
     }
 
-    void merge(const Quantile &other)
+    void merge(const Quantile &other, bool aggregate)
     {
-        _quantile.merge(other._quantile);
+        if (aggregate && !_quantile.is_empty()) {
+            if (other._quantile.is_empty()) {
+                return;
+            }
+            const uint32_t num_quantiles = 10;
+            auto quantiles = _quantile.get_quantiles(num_quantiles);
```

Comment:
I'm not sure what passing 10 here does? Why wouldn't we want to use the same quantiles we use in output - 50, 90, 95, 99?

Reply:
I thought I little about this...
I'm generating a new quantile variable with the sum of current and other quantiles. The problem of only getting 4 values would be the displayed below:

![image](https://user-images.githubusercontent.com/23251360/189718498-8fbcbfd7-f646-494b-adb9-97e8caab0ae3.png)

Reply:
Ah right, the output of a sum aggregation across quantiles actually isn't another quantile - it's a vector of individual results at each percentile (so like, an array of four numbers). Would that work in this design?

Reply:
You are right, That's why I got the 10 quantiles for each and sum them... so we have 10 values that were sum... So when we output the 0.5 0.9 0.95 0.99 it would be a more precise value 

The return of this merge is a quantile structure with 10 data point in which each data point is a sum of quantile the merge quantiles

so:
new_quantile[0] = q0.1 + other_q0.1
new_quantile[1] = q0.2 + other_q0.2
....
new_quantile[9] = q1.0 + other_q1.0

Reply:
proper add the sum values